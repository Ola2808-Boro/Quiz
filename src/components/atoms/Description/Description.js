import styled from "styled-components";
import { device } from "helpers/breakpoints";

export const Description = styled.p`
  font-size: ${({ theme }) => theme.fontSize.l};
  color: ${({ theme }) => theme.colors.text};
  letter-spacing: 1px;

  @media ${device.mobileS} {
    font-size: ${({ theme }) => theme.fontSize.m};
  }
  @media ${device.mobileM} {
    font-size: ${({ theme }) => theme.fontSize.l};
  }
  @media ${device.mobileL} {
    font-size: ${({ theme }) => theme.fontSize.l};
  }
  @media ${device.tablet} {
    font-size: ${({ theme }) => theme.fontSize.xxxl};
  }
  @media ${device.laptop} {
    font-size: ${({ theme }) => theme.fontSize.xl};
  }
  @media ${device.laptopL} {
    font-size: ${({ theme }) => theme.fontSize.xxl};
  }
`;
