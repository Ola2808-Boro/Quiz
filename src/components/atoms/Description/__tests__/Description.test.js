import rerender from "react-test-renderer";
import "jest-styled-components";
import React from "react";
import AppProviders from "providers/AppProvider";
import { Description } from "components/atoms/Description/Description";

describe("Test description component", () => {
  it("default style", () => {
    const tree = rerender
      .create(
        <AppProviders>
          <Description />
        </AppProviders>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
