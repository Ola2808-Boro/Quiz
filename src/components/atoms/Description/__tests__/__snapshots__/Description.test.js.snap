// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test description component default style 1`] = `
<p
  className="hSSqPr"
/>
`;
